scripts/migration/compareBalances.js

```diff
@@ -0,0 +1,33 @@
+const JSBI = require('jsbi');
+const ethers = require('ethers');
+const fs = require('fs');
+const csv = require('csv-parser');
+const { getBalance } = require('./helpers');
+
+const nodeUrl = "http://node1.testnet.leapdao.org:8645";
```

Comment:
always use load balancer to talk to testnet/mainnet. For testnet it is 
```
https://testnet-node.leapdao.org
``` 

(`https://testnet-node1.leapdao.org` also works but discouraged because of confusion)

Reply:
domains `node1.testnet.leapdao.org` and `node2.testnet.leapdao.org` are not supported anymore (it is coincidence that node1 still points to testnet sentinel)

Reply:
Let's use `process.env` and remove the default? 😄

Reply:
Yeah, thought so as well. Now getting URL from `process.argv[2]`. Still left default to `https://testnet-node.leapdao.org`